setup.py

```diff
@@ -443,6 +443,7 @@ def load_fixtures(configure):
     res.append(u'fixtures/sites_site.json')
     res.append(u'fixtures/auth_user.json')
     res.append(u'fixtures/socialaccount_socialapp.json')
+    res.append(u'fixtures/redirects_redirect.json')
```

Comment:
@martinmacko47 V pripade ze databaza uz existuje tak sa nezavolju fixtures. Je potrebne aby si si zavolal prikaz
```
env/bin/python manage.py loaddata fixtures/redirects_redirect.json
```
Je to vyhovujuce alebo to treba prerobit?

Reply:
@viliambalaz Spravime to rovnako ako na tom druhom projekte (vid slack). Tam sa `load_redirects` v setupe vola vzdy, aj ak uz DB bola vytvorena.

Reply:
:ok: